ts/initializeTodos.ts

```diff
@@ -0,0 +1,57 @@
+import { todos } from "./todoapp.js";
+import { storage } from "./storage.js";
+import { Todo } from "./types.js";
+
+const activeTab = document.querySelector("#active-button") as HTMLElement;
+const completedTab = document.querySelector("#completed-button") as HTMLElement;
+
+function initializeTodos() {
+  const allCheckBoxes = getCheckBoxes();
+  allCheckBoxes.forEach((checkBox) => {
+    const currentTodo = checkBox.closest(".todo-ul-item") as HTMLElement;
+    const currentTodoId = currentTodo.id;
+    const label = currentTodo.firstElementChild as HTMLElement;
+    todos.forEach((todo) => {
+      if (todo.id !== currentTodoId) return;
+      if (todo.completed) {
+        checkBox.checked = true;
+        label.style.textDecoration = "line-through";
+      } else {
+        checkBox.checked = false;
+      }
+    });
+
+    checkBox.addEventListener("click", () => {
+      const isActiveTab = activeTab.classList.contains("active");
+      const isCompletedTab = completedTab.classList.contains("active");
+      const shouldDisplay =
```

Comment:
I tried to remove this logic and test your app, and I believe this is dead logic, probably you don't need this code